core/src/main/java/org/mule/internal/connection/PooledConnectionProviderWrapper.java

```diff
@@ -38,8 +41,24 @@
      */
     public PooledConnectionProviderWrapper(ConnectionProvider<Config, Connection> delegate, PoolingProfile poolingProfile)
```

Comment:
why this constructor with default value? This is pretty internal for masking being a neccesity